@ndlib/gatsby-theme-marble/src/components/Shared/DropDown/index.js

```diff
@@ -23,13 +23,15 @@ const DropDown = ({ sxStyle, sxTiny, buttonLabel, options }) => {
       <Button
         variant='primary'
         className='dropdown-toggle'
+        aria-label='Save to a Portfolio'
```

Comment:
Could this be `{ buttonLabel }`?

Reply:
I tried using `aria-label={buttonLabel}` but it's an object and this needs to be a string.
![Screen Shot 2021-08-12 at 9 29 22 AM](https://user-images.githubusercontent.com/56038074/129205335-4048996c-fd85-4fbf-b743-cbef473bc417.png)